### ISSUE TITLE
g8 command documentation issue for Windows in Quick Start section of tutorial

### ISSUE THREAD

**Issue by brinklec** [http4s/http4s#1169]
The command given in Quick Start to clone from the g8 template works as is on Linux, but to get it to work in Windows I have to replace the two arguments "-sbt-version 0.13.13" given in the tutorial with a single argument "-sbt-version0.13.13" (omitting the space between what are separate arguments in Linux).

**Comment by rossabaker** [comment]
How did you install sbt on Windows?  Is it a difference between some batch file and the standard Linux launcher, I wonder?  I don't have Windows myself to try this, but this is interesting.

**Comment by brinklec** [comment]
This is on a corporate laptop managed by my IT department, so I didn’t install sbt myself.  I gave them the sbt download URL, but I’m not sure what procedure they followed.  I do have another Windows laptop of my own, and could do some testing with that and report back by next week.

**Comment by brinklec** [comment]
My sincere apologies, had a momentary lapse of memory, and what I said is not true of the current install.  This one I downloaded myself into a personal area.  It is the 0.13.13 launcher.

From: Chris Brinkley
Sent: Saturday, May 06, 2017 12:36 AM
To: 'http4s/http4s' <reply@reply.github.com>
Subject: RE: [http4s/http4s] g8 command documentation issue for Windows in Quick Start section of tutorial (#1169)

This is on a corporate laptop managed by my IT department, so I didn’t install sbt myself.  I gave them the sbt download URL, but I’m not sure what procedure they followed.  I do have another Windows laptop of my own, and could do some testing with that and report back by next week.
